src/main/java/co/inventorsoft/springmail/service/Delivery.java

```diff
@@ -0,0 +1,66 @@
+package co.inventorsoft.springmail.service;
+
+import co.inventorsoft.springmail.dao.EmailDAO;
+import co.inventorsoft.springmail.model.Email;
+import lombok.AllArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.mail.MailException;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+@Service
+@AllArgsConstructor
+public class Delivery {
+
+    private EmailDAO emailDAO;
+
+    private JavaMailSender mailSender;
+
+    @Scheduled(fixedRate  = 60000) //1 min
+    public void deliverEmails(){
+        new EmailDeliver().start();
```

Comment:
It's good that you're thinking about sending emails in background, but methods that are annotated by @Scheduled are already running in separate threads, so you don't need to manage threads manually